index.html

```diff
@@ -2532,6 +2532,22 @@ <h3>Conformance of <a>DAPT Documents</a></h3>
         <a href="https://www.w3.org/TR/ttml2/#profile">profile</a> vocabulary of [[TTML2]] in the
         <dfn data-cite="ttml2#terms-content-profile">content profile</dfn> defined
         in <a href="#profiles"></a>.</p>
+      <aside class="note">The provisions in this specification are expressed in terms of TTML2 processor types,
```

Comment:
Shouldn't the new note be in §7.2 "Conformance of DAPT Processors" rather than in the section on document conformance?

Reply:
Yes, my mistake. Will update.